test/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "#": "We must put jest in a sub-directory otherwise VS Code somehow picks up",
```

Comment:
Half the comment went missing! I suspect this may be because both of them used the `"#"` key.

Reply:
Weird, good catch!